examples/quickstart_etl/README.md

```diff
@@ -0,0 +1,219 @@
+
+```bash
+dagster project from-example --name my-dagster-project --example quickstart_etl
+```
+
+First, install your Dagster repository as a Python package. By using the --editable flag, pip will install your repository in ["editable mode"](https://pip.pypa.io/en/latest/topics/local-project-installs/#editable-installs) so that as you develop, local code changes will automatically apply.
+
+```bash
+pip install -e ".[dev]"
+```
+
+Then, start the Dagit web server:
+
+```bash
+dagit
+```
+
+Open http://localhost:3000 with your browser to see the project.
+
+You can start writing assets in `quickstart_etl/assets/`. The assets are automatically loaded into the Dagster repository as you define them.
+
+<!-- ### Option 3: Running it in Gitpod
+Open source friendly
+<TODO: set up gitpod> -->
+
+## Step 1: Materializing assets
+
+An asset is a software object that models a data asset, which can be a file in your filesystem, a table in a database, or a data report.
+
+Navigate to `hackernews` asset group. You will find four assets with different tags in the group:
+* `hackernews_topstory_ids` fetches a list of top story ids from a HackerNews endpoint
+* `hackernews_topstories` takes the list of ids and pulls the story details from HackerNews based on the ids.
+* `hackernews_stories_by_date` aggregates stories by date, which later will be used to generate a metrics report.
+* `hackernews_stories_word_cloud` explores the dataset by visualizing the word cloud of trending story titles.
+
+<!-- V0 -->
+<!-- <p align="center">
+    <img height="500" src="https://user-images.githubusercontent.com/4531914/200672470-fe8c7880-6fb7-4ec7-b5a6-c167416cbf1c.png">
+</p> -->
+
+
+<p align="center">
+    <img height="500" src="https://user-images.githubusercontent.com/4531914/201776745-4aeb3e2c-bc45-425d-9bec-c49166eb7b05.png">
+</p>
+
+Click `Materialize all`. Now you've launched a Dagster run and it will materialize all the assets in this group. *(It may take 1-2 minutes to fetch all top 500 stories from HackerNews).*
+
+## Step 2: Viewing assets
+
+Once the run finishes, go to <TODO: relative url?>. You can now find a preview of the asset you just materialized.
+
+<TODO: point out dataframe preview. explain asset metadata>
+
+<p align="center">
+    <img height="500" src="https://user-images.githubusercontent.com/4531914/201304011-0c62ad06-36d8-4984-9278-b3b670254da3.png">
+</p>
+
+
+<TODO: explain word cloud plot 👇 . need to find a cloud friendly way to view the plot>
+
+<p align="center">
+    <img height="500" src="https://user-images.githubusercontent.com/4531914/201312310-44fb053c-9352-498d-9c23-ab4d7be6afec.png">
+</p>
+
+## Step 3: Configuring data pipelines
+
+You've got your first couple of assets generated. Let's add one more data source to the pipeline: GitHub.
+
+Navigate to the `github` asset group and click `Materialize all`. When you try to materialize assets in this group, you may encounter the following error:
```

Comment:
you *will* encounter this error, and that's because we need to add some secrets!

Also, I think we should call out that there are four options for how a user could add secrets here: 

1. GH PAT token
2. Adding a secret via the terminal
3. Adding a secret via Dagster's Cloud Env Vars
4. Adding secrets via a 3rd party secrets manager <- we should include this as an option, right?

Reply:
i ended up simplifying this quickstart a lot and made it that 1) it focuses on dagster basics and aims to show the real value (plot in asset metadata) within the first run. 2) it now doesn't interact with env vars, so we'll include the env var section in our quickstarts (snowflake, dbt, etc - all other ones) instead.